packages/framework/fluid-static/api-report/fluid-static.api.md

```diff
@@ -15,6 +15,9 @@ import { IFluidLoadable } from '@fluidframework/core-interfaces';
 import { IRuntimeFactory } from '@fluidframework/container-definitions/internal';
 import { ISharedObjectKind } from '@fluidframework/shared-object-base';
 
+// @public
+export type CompatMode = "1" | "2";
```

Comment:
Maybe a long term question, but the purpose here is to call out Supported Compat Modes for given release version? When we release 3.x I would assume compat mode options become 2 and 3?

Reply:
I personally don't think this is a sustainable approach long term so I hope we have a better solution for 3?  But if not, then yes this would become `"2" | "3"` in our 3.0 release to reflect that it only supports N - 1 interop.